packages/rspeedy/create-rspeedy/template-react-ts/src/rspeedy-env.d.ts

```diff
@@ -1,6 +1,7 @@
 /// <reference types="@lynx-js/rspeedy/client" />
 
 declare module '@lynx-js/types' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-object-type
```

Comment:
It looks like we are missing this in `packages/rspeedy/create-rspeedy/template-react-vitest-rltl-ts/src/rspeedy-env.d.ts`. Perhaps we could move it to `template-common`.